app/uk/gov/hmrc/exports/scheduler/jobs/PurgeAncientSubmissionsJob.scala

```diff
@@ -35,23 +35,18 @@ class PurgeAncientSubmissionsJob @Inject() (
   submissionRepository: SubmissionRepository,
   transactionalOps: PurgeSubmissionsTransactionalOps,
   jobRunRepository: JobRunRepository
-)(implicit ec: ExecutionContext)
+)(implicit @Named("backgroundTasksExecutionContext") ec: ExecutionContext)
     extends ScheduledJob with Logging {
 
-  override val name: String = "PurgeAncientSubmissions"
-
-  override def interval: FiniteDuration = appConfig.purgeAncientSubmissions.interval
-
   override def firstRunTime: Option[LocalTime] = Some(appConfig.purgeAncientSubmissions.elapseTime)
+  override def interval: FiniteDuration = appConfig.purgeAncientSubmissions.interval
 
-  lazy val expiryDate = {
+  lazy val expiryDate: ZonedDateTime = {
```

Comment:
As this class is a singleton, and this date is a val, then this date will only ever change when the application is redeployed. So that means if we don't redeploy for a week then submissions will start to build up that are older than the 180 days! I think this date (and the filter) need to be methods rather than vals :thinking: